doc/mgr/dashboard.rst

```diff
@@ -133,6 +133,18 @@ dashboard.
 
 .. _dashboard-landing-page-status:
 
+Switch Dashboard Landing Page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are two different views available for the landing page of the Dashboard.
+
+You can toggle between them from the manager modules page. At: ``Cluster >> Manager Modules >> Dashboard >> Edit``.
+Editing the ``FEATURE_TOGGLE_DASHBOARD`` option will change the landing page, from one view to another.
+
+By default, the option is activated, displaying a view like so:
+
+.. image:: dashboard-landing-page.png
```

Comment:
Since we intend this Dashboard v3 to become the new & official, I'd avoid talking about it as a transitional thing as these docs might last for many releases, and instead present it as just the "Dashboard Landing Page" (you may leave the "version 3", or "iteration #3", whatever, ... to make readers/users aware of it), and only in the bottom of the section you can add a "Note/warning box" explaining users how to rollback to the previous one and that it'll be disabled in future releases.